src/main/java/com/targomo/client/api/pojo/CompetingRoutingOption.java

```diff
@@ -25,4 +27,9 @@ public CompetingRoutingOption(@JsonProperty("routingOptions") RoutingOptions rou
         this.routingValueMultiplier = routingValueMultiplier == null ? 1.0f : routingValueMultiplier;
         this.routingAggregationType = routingAggregationType == null ? RoutingAggregationType.MIN :routingAggregationType;
     }
+
+    @Override //overriding hash code since the has would change with each execution for enums (routingAggregationType)
+    public int hashCode(){
+        return Objects.hash(routingOptions, routingValueOffset, routingValueMultiplier, routingAggregationType.toString());
```

Comment:
if you want to include the enum in the hash you can use `routingAggregationType.ordinal()`. the hash of the ordinal doesn't change at each execution!